plugins/parsers/registry.go

```diff
@@ -141,6 +142,10 @@ type Config struct {
 	CSVTimestampColumn   string   `toml:"csv_timestamp_column"`
 	CSVTimestampFormat   string   `toml:"csv_timestamp_format"`
 	CSVTrimSpace         bool     `toml:"csv_trim_space"`
+
+	// FormData configuration
+	FormDataTagKeys     []string `toml:"form_data_tag_keys"`
+	FormDataAllowedKeys []string `toml:"form_data_allowed_keys"`
```

Comment:
I don't think we will need the `form_data_allowed_keys` option, one could use the fielddrop/fielddrop options instead.